src/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect, useRef } from 'react';
+// UNFORTUNATELY WHEN LIGHT VERSION IS USED, SOME ANIMATIONS ARE NOT WORKING AS EXPECTED
+// import lottie from 'lottie-web/build/player/lottie_light';
+import lottie from 'lottie-web';
+import type { AnimationItem, AnimationConfigWithData, AnimationConfig } from 'lottie-web';
+
+import { getEffect, getEvents } from './utils';
+import type { Animation, AnimationEffect } from './utils';
+
+type Props = {
+  animation: Animation;
+  reverse?: boolean;
+  strokeColor?: string;
+  pathCss?: string;
+  options?: Partial<AnimationConfig>;
+  size?: number;
+  loop?: AnimationConfig['loop'];
+  autoplay?: AnimationConfig['autoplay'];
+  wrapperStyle?: React.CSSProperties;
+} & React.HTMLProps<HTMLDivElement>;
+
+const UseAnimations: React.FC<Props> = ({
+  animation: { animationData, animationKey },
+  reverse = false,
+  size = 24,
+  strokeColor,
+  pathCss,
+  loop,
+  autoplay,
+  wrapperStyle,
+  options,
+  onClick,
+  ...other
+}) => {
+  const [animation, setAnimation] = useState<AnimationItem>();
+  const ref = useRef<HTMLDivElement>(null);
+  const getRandomId = (key: Animation['animationKey']) =>
+    `${key}_i${Math.floor(Math.random() * 100 + 1)}`;
+
+  useEffect(() => {
+    const animEffect: AnimationEffect = getEffect(animationKey);
+    const animationId = getRandomId(animationKey);
+
+    if (strokeColor || pathCss) {
+      const css = `#${animationId} path { stroke: ${strokeColor || 'inherit'}; ${pathCss}}`;
```

Comment:
the condition is `strokeColor || pathCss` so if `strokeColor` is `undefined` the css would work fine?

Reply:
yop, if there's no need to define some special color, there's no need for our custom css in the `head` and color will be black.